config/locales/en.yml

```diff
@@ -1503,9 +1503,14 @@ en:
     title: "Notifications"
     email_settings: "E-mail notifications"
     form:
-      assignments: "Assignments notifications"
-      recent_notification: "Change notifications"
-      system_message: "Update notifications"
+      assignments: "Assignment"
+      notification_scinote: 'Show in sciNote'
+      notification_email: 'Notify me via email'
+      assignments_description: 'Assignment notifications appear whenever you get assigned to a team, project, task.'
+      recent_notification: 'Recent changes'
+      recent_notification_description: 'Recent changes notifications appear whenever there ia a change on a task you are assigned to.'
+      system_message: 'System message'
+      system_message_description: 'System message notifications appear whenever there is new release of sciNote'
```

Comment:
Change this to `System message notifications are specifically sent by site maintainers to notify all users about a system update.`

You have my blessing to ignore the issue's text, and use mine 😸 .